styles/pages/about.module.scss

```diff
@@ -39,12 +38,14 @@ $partner-logo-margin-right: 30px 30px 60px 36px 36px;
   }
 
   &__text {
-    &--margin-top-small {
-      margin-top: 30px;
-    }
+    &--margin-top {
+      &--small {
```

Comment:
one dash (here and fore `medium`) ?

Reply:
https://github.com/babajka/babajka-frontend/pull/94#discussion_r392601811